otx/cli/utils/importing.py

```diff
@@ -52,8 +62,35 @@ def get_backbone_list(backend):
     raise ValueError(f"{backend} cannot be imported.")
 
 
+def get_available_backbone_list(backend):
+    """Gather available backbone list from json file."""
+    available_backbone_path = os.path.join(get_otx_root_path(), "cli/builder/supported_backbone")
+    available_backbone_path = os.path.join(available_backbone_path, f"{backend}.json")
+    available_backbones = {}
+    if os.path.exists(available_backbone_path):
+        with open(available_backbone_path, "r", encoding="UTF-8") as f:
+            available_backbones = json.load(f)
+        available_backbones = available_backbones["backbones"]
```

Comment:
mmcls, mmdet, mmseg backbones don't need `required` or `options` or `available` option?

Reply:
All mmcls, mmdet, and mmseg backbones use corresponding `options`, and `required` are used for both find and build. This is already created in the form of a json file.

Reply:
Ah, I missed that, Thanks